### PR TITLE
Revised r2modman profile sharing

### PR DESCRIPTION
**Pull request by Boxofbiscuits97**
## Profile Sharing
Pertains to issue #6  and #27 

Just a cleaned up version to be easily used on the wiki following the style of the other guides.

### CODE REVIEW

**Comment by ghotato** [comment]
thanks a bunch!!

**Comment by ghotato** [comment]
like this way better than mine, sweet and simple.

File: docs/user-guide/installation/installing-r2modman.md [hunk 1]
```diff
@@ -55,4 +55,29 @@ Once you have succesfully installed r2modman, you can finally install mods.
...
+
+1. Select **"Export profile as code"** in the **profile tab** in settings of your profile.
+
+![Exporting Profile](../docs/files/r2modman-install/exportprofilecode.png)
+
+This will copy a code you can send to friends to your clipboard automatically.
+
+### Importing a Profile
+
+1. Click **"Import/Update"** in the **Profile selction** area after selecting Lethal Company in r2modman.
```
Comment (legoandmars) [inline docs/user-guide/installation/installing-r2modman.md:74]: typo in selection

**Review by legoandmars — CHANGES_REQUESTED** [review]

**Comment by legoandmars** [comment]
I do think this would be better as a separate page, but #29 changes how file structure works, so adding the page would have to be redone anyways

Once #29 is merged I'll probably add a bit more detail and separate this into a new page myself

in the meantime i'll just merge this as-is once the typo is fixed

**Pull request closed without merging**